globals.json

```diff
@@ -1194,8 +1195,7 @@
 		"ScriptEngineMinorVersion": true,
 		"VBArray": true,
 		"WScript": true,
-		"WSH": true,
-		"XDomainRequest": true
```

Comment:
Not worth doing a major release just to remove this. Open an issue about removing it and I'll do it when we do the next major release.

Reply:
Sure thing.  Moved to #155.